.github/workflows/list-changed-charts.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" || exit; pwd;)"
+
+cd ${script_dir}/../../ # cd to the root of the repo
+
+CHARTS=("swan" "swan-cern")
+
+for CHART_NAME in "${CHARTS[@]}"; do
+
+    # Determine last tag on the chart
+    LAST_CHART_TAG="$(git tag --list --sort -version:refname ${CHART_NAME}@* | head -n 1)"
+
+    # Determine if there are any changes on the chart/ folder since last tag
+    HAS_CHANGED="$(git diff --quiet HEAD ${LAST_CHART_TAG} -- ./${CHART_NAME} || echo true)"
+
+    # print output
+    echo "${CHART_NAME}_has_changed=${HAS_CHANGED}"
```

Comment:
You could make this a conditional and just output e.g. `swan_has_changed` if the result of the diff is 1, else not print at all.

Reply:
The output of this script is piped to `>> $GITHUB_OUTPUT` to set an [output for the step](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter) which is mapped to the [job output](https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs). This needs output in key=value format

Reply:
Ok, it's just that if there are no changes, you emit "swan_has_changed=". That just assigns a null value to the key and it is fine?

Reply:
Fixed this, now it prints =false or =true, previously I was never checking for false